src/test/scala/org/broadinstitute/dsde/agora/server/webservice/PermissionIntegrationSpec.scala

```diff
@@ -246,4 +249,48 @@ class PermissionIntegrationSpec extends FlatSpec with RouteTest with ScalatestRo
       }
   }
 
+  "Agora" should "successfully list permissions for multiple methods simultaneously" in {
+
+    val payload:Seq[AgoraEntity] = Seq(
+      AgoraEntity(agoraEntity1.namespace, agoraEntity1.name, agoraEntity1.snapshotId),
+      AgoraEntity(agoraEntity2.namespace, agoraEntity2.name, agoraEntity2.snapshotId),
+      AgoraEntity(agoraEntity1.namespace, agoraEntity1.name, Some(12345))
+    )
+
+    Post(ApiUtil.Methods.withLeadingVersion + "/permissions", payload) ~>
+      methodsService.multiEntityPermissionsRoute ~>
+      check {
+        assert(status == OK)
+        val entityAclList = responseAs[Seq[EntityAccessControl]]
+        assertResult(3) {entityAclList.size}
+
+        // check first - should get permissions
+        {
+          val stubEntity = AgoraEntity(agoraEntity1.namespace, agoraEntity1.name, agoraEntity1.snapshotId)
+          val found = entityAclList.find(_.entity == stubEntity)
+          assert(found.isDefined)
+          assert(found.get.message.isEmpty)
+          assert(found.get.acls.size == 1)
+          assert(found.get.acls.head.user == mockAuthenticatedOwner.get)
+          assert(found.get.acls.head.roles.canManage)
+        }
+        // check second - it exists, but we don't have permissions to see it
+        {
+          val stubEntity = AgoraEntity(agoraEntity2.namespace, agoraEntity2.name, agoraEntity2.snapshotId)
+          val found = entityAclList.find(_.entity == stubEntity)
+          assert(found.isDefined)
+          assert(found.get.message.nonEmpty)
+          assert(found.get.acls.isEmpty)
+        }
+        // check third - it doesn't exist in the db
```

Comment:
I've added a redacted entity to the test. Under the covers, redaction removes all permissions from the entity, so it's actually the same as the second test - it exists in the db, but we're not allowed to see it.